### DISCUSSION TITLE
Availability at Homebrew?

### DISCUSSION THREAD

**Discussion by mfelipe** [ohmyzsh/ohmyzsh#12787] [Ideas]
Hi all! First, I wanted to say that I've been using ohmyzsh for about 2 years and I'm very pleased.

It probably has been discussed before and I'm sorry if it's a repetition, but: are there any chance of making it available via brew?

Cheers!

**Comment by carlosala** [comment]
It'd be great if some homebrew maintainer packs it! Anyways, our installation method is fairly simple, I don't think it's a blocker for new users.
Thanks for the idea!
